XCUITests/FxScreenGraph.swift

```diff
@@ -709,6 +719,13 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
     }
 
     map.addScreenState(AddCustomSearchSettings) { screenState in
+        screenState.gesture(forAction: Action.AddCustomSearchEngine) { userState in
+            app.tables.textViews["customEngineTitle"].staticTexts["Search Engine"].tap()
+            app.typeText("Mozilla Engine")
+            app.tables.textViews["customEngineUrl"].staticTexts["URL (Replace Query with %s)"].tap()
+            app.typeText("https://developer.mozilla.org/search?q=%s")
```

Comment:
I don't get this typeText working...there is this workaround instead that we could use: 
```
           UIPasteboard.general.string = "https://developer.mozilla.org/search?q=%s"
            
            let tablesQuery = app.tables
            let customengineurlTextView = tablesQuery.textViews["customEngineUrl"]
            customengineurlTextView.staticTexts["URL (Replace Query with %s)"].tap()
            customengineurlTextView.press(forDuration: 1.0)
            app.staticTexts["Paste"].tap()
```
In this case, the tap on Save button has to be moved to the tests otherwise there was a failure trying to tap there (in the graph we can't add a waitFor, and so the only solution was to add sleep or move the tap on Save button to the test)